client/src/components/DataFiles/DataFilesProjectsList/DataFilesProjectsList.module.scss

```diff
@@ -0,0 +1,4 @@
+.root {
+  flex-grow: 1;
+  display: flex;
+}
```

Comment:
The column headers and the text in the projects listing should be 16px to match the other areas. Can we make that so here?